src/sharepoint/rest/search.ts

```diff
@@ -244,10 +244,19 @@ export class Search extends QueryableInstance {
     }
 
     /**
-     * .......
+     *
      * @returns Promise
      */
-    public execute(query: SearchQuery): Promise<SearchResults> {
+    public execute(query?: SearchQuery): Promise<SearchResults> {
+
+        // Used with alternative search
+        if (typeof query === "undefined" && typeof this.searchQuery !== "undefined") {
+            if (typeof this.pageNumValue !== "undefined") {
```

Comment:
Do the defaults here match the OOB defaults when hitting the search endpoint without specifying values?